src/crypto/tls/handshake_client.go

```diff
@@ -186,6 +186,9 @@ func (c *Conn) clientHandshake() (err error) {
 		return err
 	}
 
+	handshakeTimings := createTLS13ClientHandshakeTimingInfo(c.config.Time)
```

Comment:
`handshakeTimings` should be instantiated at the very beginning of `clientHandshake()`.